components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/operation/mgt/ConfigOperation.java

```diff
@@ -39,7 +40,7 @@ public void addConfigProperty(String name, Object value, Class<?> type) {
         properties.add(new Property(name, value, type));
     }
 
-    public class Property {
+    public class Property implements Serializable{
```

Comment:
Please fix formatting issues in this particular line. There needs to be a space right after the token "Serializable".